src/row.rs

```diff
@@ -186,6 +186,18 @@ pub struct Row<'stmt> {
     pub(crate) stmt: &'stmt Statement<'stmt>,
 }
 
+impl<'stmt> fmt::Debug for Row<'stmt> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let mut row = Vec::new();
```

Comment:
This (and going right through Value instead of e.g. ValueRef) is a bit wasteful but I guess it wouldn't matter for a Debug impl anyway.